src/pugixml.hpp

```diff
@@ -312,6 +312,8 @@ namespace pugi
 		It begin() const { return _begin; }
 		It end() const { return _end; }
 
+		bool empty() const { return _begin == _end; };
```

Comment:
Nitpicking ,extra, probably unwanted, ```;``` at the end of the line
```suggestion
		bool empty() const { return _begin == _end; }
```

Reply:
Fixed.